packages/wonder-blocks-date-picker/src/components/date-picker.tsx

```diff
@@ -2,17 +2,33 @@ import {StyleSheet} from "aphrodite";
 import {Temporal} from "temporal-polyfill";
 import * as React from "react";
 import {DayPicker} from "react-day-picker";
-import {enUS} from "react-day-picker/locale";
+import {enUS, type Locale} from "react-day-picker/locale";
 
 import {View, type StyleType} from "@khanacademy/wonder-blocks-core";
 import {semanticColor, sizing} from "@khanacademy/wonder-blocks-tokens";
-import {TemporalLocaleUtils} from "../util/temporal-locale-utils";
+import {
+    TemporalLocaleUtils,
+    enUSLocaleCode,
+} from "../util/temporal-locale-utils";
 import type {CustomModifiers} from "../util/types";
 
 import DatePickerInput from "./date-picker-input";
 import DatePickerOverlay from "./date-picker-overlay";
 
 interface Props {
+    /**
+     * The locale to use for the dates: a string name matching a Locale object
+     * imported from react-day-picker.
+     * If not provided, it will fall back to enUS.
+     */
+    locale?: Locale;
+    /**
+     * Optional writing direction to pass in as a prop.
+
+     * Useful for overlays that render outside of normal tree with `dir="rtl"`,
+     * such as in consumer stories.
+     */
+    dir?: "ltr" | "rtl";
```

Comment:
I wonder if `dir` should be determined programmatically using something like [closest()](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest) to see if an ancestor has the dir attribute set. Or if we should start adding the `dir` prop to all components. What do you think?